host_applications/linux/apps/raspicam/imx219_modes.h

```diff
@@ -87,7 +87,7 @@ struct sensor_regs imx219_8MPix[] =
       {0x0309, 0x0A},      // OPPXCK_DIV [4:0]
       {0x030B, 0x01},      // OPSYCK_DIV
       {0x030C, 0x00},      // PLL_OP_MPY [10:8]
-      {0x030D, 0x55},      // PLL_OP_MPY [7:0]
+      {0x030D, 0x72},      // PLL_OP_MPY [7:0]
```

Comment:
That's probably the key one. I don't know where the original post got that value from as I can't see it ever being sent as 0x55 by the firmware.